app/javascript/packs/components/SocialDistanceNotice.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import Box from "@material-ui/core/Box";
+import Container from "@material-ui/core/Container";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.secondary.lightGray,
+  },
+  heading: {
+    color: theme.palette.primary.main,
+    fontWeight: "600",
+    letterSpacing: theme.spacing(0.03),
+    paddingTop: theme.spacing(3),
+    paddingBottom: theme.spacing(3),
+  },
+  message: {
+    lineHeight: "1.5rem",
+    paddingBottom: theme.spacing(3),
+    paddingLeft: theme.spacing(8),
+    paddingRight: theme.spacing(8),
+  },
+  cdcLink: {
+    color: theme.palette.text.primary,
+    fontWeight: "bold",
+  },
+}));
+
+const SocialDistanceNotice = () => {
+  const classes = useStyles();
+  return (
+    <Container maxWidth="lg">
+      <Box className={classes.root}>
+        <Typography align="center" variant="body1" className={classes.heading}>
+          NOTICE FOR VOLUNTEERS
+        </Typography>
+
+        <Typography align="center" variant="body1" className={classes.message}>
+          <b>
+            Please follow all COVID-19 guidance from federal, state, and local
+            officials.
+            <br />
+          </b>
+          Make sure to maintain social/physical distancing and safety guidelines
+          when fulfilling
+          <br />
+          requests. To learn more, visit the{" "}
```

Comment:
I'd recommend letting the sentence break naturally and specify a width rather than forcing the desired break. Forcing breaks can lead to noticeably short line lengths on mobile.